packages/ckeditor5-ui/src/editorui/bodycollection.ts

```diff
@@ -82,7 +82,7 @@ export default class BodyCollection extends ViewCollection {
 			children: this
 		} ).render() as HTMLElement;
 
-		let wrapper = document.querySelector( '.ck-body-wrapper' );
+		let wrapper = Array.from( document.body.children ).find( el => el.classList.contains( 'ck-body-wrapper' ) );
```

Comment:
We are aware that some integrators move the `.ck-body` to other places in DOM to achieve some specific side-effects, for instance, make all editor balloons and floatables clipped by some arbitrary ancestor with overflow, etc. (they wrap the editor instance and `.ck-body` in a div with `overflow: hidden`).

Knowing this, we should not assume the `.ck-body` wrapper is the child of `<body>`. Doing that may cause some regressions for these integrators (and customers). What we can do though, is keep the static reference to the wrapper somewhere in `BodyCollection` and make it a kind of singleton (since it already acts as one, at least in this regard). This will be as fast as possible, anyway.

Reply:
@oleq is the latest commit I pushed (rebased) along the lines of what you're thinking?